laravel/the-basics/databases.html.md.erb

```diff
@@ -515,8 +510,9 @@ To test whether your Laravel Fly App can successfully connect with your Upstash
 <li>Using `REDIS_URL` should override `REDIS_HOST`, add here the `REDIS_URL` received during `fly redis create`</li>
 <li>Upstash only supports "database 0", so update your `REDIS_CACHE_DB` config to 0. Otherwise, you will receive an error : "SELECT` failed: ERR Only 0th database is supported! Selected DB: 1..."</li>
 </ul>
-
-2) Then deploy your `fly.toml` changes:
+</li>
+<li>
+Then deploy your `fly.toml` changes:
```

Comment:
Then please make sure to add the third item as part of the list. Starts in line 520:
```
<li>Once deployment completes... 
```
And ends is line 533 with the closing tag for that last item and the list itself: 
```
</li></ol>
```
Otherwise the third item here won't be part of the list and the next section as well will be included, likeso:
_____________________________________________
![image](https://user-images.githubusercontent.com/33826441/211872204-cb69e59d-c402-4e81-be7e-951b20b41665.png)

Reply:
Hi @jeregrine ! Check line 522, the third item needs to be enclosed in <li> tags

The parent <ol> also needs to be closed, otherwise the next sections get grouped under the list